### ISSUE TITLE
Cannot Reproduce the Reported Accuracy on RoboSpatial-Home Benchmark

### ISSUE THREAD

**Issue by tyb197** [chanhee-luke/RoboSpatial-Eval#3]
Hi! Thanks for your excellent work and open-sourcing evaluation code! I’m trying to reproduce the RoboSpatial-Home benchmark results using RoboPoint (initialized from wentaoyuan/robopoint-v1-vicuna-v1.5-13b) and LLaVA-Next (initialized from lmmslab/llama3-llava-next-8b on Hugging Face), but my accuracy metrics differ significantly from your paper. 

**My results:**

```
**RoboPoint**
"category_stats": {
    "context": {
      "accuracy": 19.672131147540984,
      "num_correct": 24,
      "num_total": 122
    },
    "compatibility": {
      "accuracy": 72.38095238095238,
      "num_correct": 76,
      "num_total": 105
    },
    "configuration": {
      "accuracy": 69.91869918699187,
      "num_correct": 86,
      "num_total": 123
    }
}

**LLaVA-Next**
"category_stats": {
    "context": {
      "accuracy": 0.0,
      "num_correct": 0,
      "num_total": 122
    },
    "compatibility": {
      "accuracy": 70.47619047619048,
      "num_correct": 74,
      "num_total": 105
    },
    "configuration": {
      "accuracy": 68.29268292682927,
      "num_correct": 84,
      "num_total": 123
    }
```

**Paper's reported metrics:**

```
**RoboPoint**
Affordance (context): 41.3
Compatibility: 36.0
Localization (configuration): 43.3

**LLaVA-Next**
Affordance (context): 9.3
Compatibility: 37.3
Localization (configuration): 48.0
```

**My questions:**
1. Are there any key hyperparameters/data preprocessing steps I might have missed? The "context" accuracy metrics (19.67 vs. 41.3 for RoboPoint and 0 vs. 9.3 for LLaVA-Next) are particularly low. However, my zero-shot evaluation results of the other two metrics are much higher than those reported in paper (even higher than those results of finetuned models) on RoboSpatial.
2. I noticed the evaluation code only uses the first predicted point for scoring in the "context" (affordance) category - Is this the intended benchmark setup? For the models predicted multi-points, this evaluation method might not be fully representative of the model's capability.

Any guidance would be greatly appreciated! Thanks again for your work.

**Comment by chanhee-luke** [comment]
Hi! Thanks for your interest :)

1. Please check table 4 of our paper for RoboSpatial-Home results! Table 3 is on our validation split, RoboSpatial-Val. It looks like your reported results match what’s on table 4.

2. Yes that is the intended setup since we use the first predicted point for robot manipulation. This is a stricter measure than how RoboPoint calculates success.

Luke

**Comment by tyb197** [comment]
I've checked the latest version of your paper and noticed I was using an older reference. Really appreciate your reply!
